standards/core/lightpush.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: 19
+title: 19/WAKU2-LIGHTPUSH
```

Comment:
No need for a number or slug. It can simply be titled WAKU2-LIGHTPUSH. Once implemented and tested, it could graduate to a Vac RFC and will get the number 19 then to indicate it replaces the current published spec.

Reply:
Ok, removed